src/BloomTests/CollectionTab/TestLibraryModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.IO;
+using Bloom;
+using Bloom.Book;
+using Bloom.Collection;
+using Bloom.CollectionTab;
+using ICSharpCode.SharpZipLib.Zip;
+using Palaso.TestUtilities;
+
+namespace BloomTests.CollectionTab
+{
+	class TestLibraryModel: LibraryModel
+	{
+		public readonly string TestFolderPath;
+
+		public TestLibraryModel(TemporaryFolder testFolder)
+			: base(testFolder.Path, new CollectionSettings(), null, new BookSelection(), new SourceCollectionsList(), null, null, new CreateFromSourceBookCommand(), null, null)
+		{
+			TestFolderPath = testFolder.Path;
+		}
+
+		private int GetDirNameOffset
+		{
+			get
+			{
+				var rootName = Path.GetFileName(TestFolderPath);
+				if (rootName == null)
+					throw new ApplicationException("Not in a test we don't!");
```

Comment:
Removed the whole conditional. It has nothing to do with the test and if it fails, the test fails, which is fine.